freewheel.podspec

```
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'freewheel'
-  s.version      = '2.1'
+  s.version      = '4'
```

```suggestion
  s.version      = '2.2'
```

On doit faire plutôt une 2.2 pour maintenir la compatibilité avec `YouboraFreewheelAdapter` (cf. https://bitbucket.org/npaw/freewheel-adapter-ios/src/883514884a0949dab01f79cc33dfb8e0ff093257/Podfile#lines-5). Ainsi on n'aura pas besoin de solliciter NPAW pour une maj de leur pod.

Je propose en parallèle de [`delete`](https://guides.cocoapods.org/terminal/commands.html#pod_trunk_delete) du pod trunk la version 3 sans quoi l'enchaînement des versions devient vraiment bizarre. Qu'en penses-tu?